src/Illuminate/Auth/Console/MakeAuthCommand.php

```diff
@@ -2,10 +2,13 @@
 
 namespace Illuminate\Auth\Console;
 
+use Illuminate\Console\AppNamespaceDetectorTrait;
```

Comment:
`use Illuminate\Console\AppNamespaceDetectorTrait;` should be under `use Illuminate\Console\Command;`